### DISCUSSION TITLE
[Quick Question] When is the next release after v0.34.0?

### DISCUSSION THREAD

**Discussion by trend-pinky-chang** [thanos-io/thanos#7121] [Questions & Answers] [answered]
Hi all,

We need the vulnerability of grpc 1.45.0 fixed as soon as possible. We saw grpc was already upgraded to 1.57.2 in master branch after the latest Thanos official build, v0.34.0. When will the next release include this fix?
https://github.com/thanos-io/thanos/pull/7078/files

Regards,
Pinky

**Comment by MichaHoffmann** [comment]
I was planning for ~ 2-3 weeks after last major release. Is that acceptable?

**Reply by trend-pinky-chang** [reply]
Since v0.34.0 was released on Jan 26, I can expect the latest release will be no later than Feb 16, right?

**Reply by nicolastakashi** [reply]
@MichaHoffmann any change we get a patch release because of this CVE?

**Reply by MichaHoffmann** [reply, marked as answer]
Oh, yeah; i can prepare one on the weekend; is that acceptable?

**Reply by trend-pinky-chang** [reply]
Thanks @MichaHoffmann , sounds great.

**Reply by MichaHoffmann** [reply]
Is there already a fix on main that needs to be cherry-picked?

**Reply by trend-pinky-chang** [reply]
This one.
https://github.com/thanos-io/thanos/pull/7078

**Reply by MichaHoffmann** [reply]
Ok, prepared a new patch release PR with the CVE fix cherry-picked: https://github.com/thanos-io/thanos/pull/7131

**Reply by MichaHoffmann** [reply]
Ok managed to finalize the release today. Sorry for the delay!
